Monika After Story/game/script-brbs.rpy

```diff
@@ -1067,3 +1067,76 @@ label monika_idle_reading_callback:
 #        m 1eka "I missed you and I'm glad you're finally back, [player]."
 #        m "I hope I don't have to wait such a long couple of minutes next time, ehehe."
 #    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="drawing_brb",
+            category=['be right back'],
+            prompt="I'm going to go draw something",
+            pool=True,
+            unlocked=True,
+
+        ),
+        markSeen=True
+    )
+
+label drawing_brb:
+    if not mas_isMoniDis(lower=True):
```

Comment:
This looks like normal+ dialogue rather than upset+

Additionally this conditional can be simplified to be more readable at a glance.
```suggestion
    if mas_isMoniNormal(higher=True):
```